deploy/crds/compliance.openshift.io_variables_crd.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
           type: string
         description:
-          description: The description of the Rule
+          description: The description of the Variable
```

Comment:
oops, looks like I forgot to make generate after the latest changes to the variables patch :-(